aiida_nanotech_empa/parsers/cp2k_gw_parser.py

```diff
@@ -0,0 +1,245 @@
+# -*- coding: utf-8 -*-
+"""AiiDA-CP2K output parser for GW calculations."""
+
+from aiida.common import OutputParsingError
+from aiida.orm import Dict
+
+from aiida_cp2k.utils import parse_cp2k_output_advanced
+from aiida_cp2k.parsers import Cp2kBaseParser
+
+HART_2_EV = 27.21138602
+
+
+def is_float(s):
```

Comment:
are integers ok in this case?

Reply:
ah integers should be fine. I agree the name is misleading, I will change it to `is_number`